src/web/hello_js/hello_js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Hello JS
+level: 1
+author: Arve Seljebu
+---
+
+# Introduksjon {.intro}
+I denne oppgaven blir du introdusert for programmeringsspråket JavaScript. Du
+skal gjøre den klassiske oppgaven *Hello world*, med en katt.
+
+![](jsbin.png)
+
+# Steg 1: Bruke JS Bin {.activity}
+Du skal bruke en nettside som heter *JS Bin* for denne oppgaven. Dette fordi
+JS Bin ser likt ut i alle nettlesere, så da er det enklere å følge stegene.
+
+## Sjekkliste {.check}
+
+- Åpne [JS Bin](http://jsbin.com/?js,console) i et eget vindu.
```

Comment:
Burde sett om det var mulig for alle lenker, implisitt, men minnes svakt at det var vanskelig med PhantomJS?